tests/core/Jalasoft.TeamUp.Resumes.Core.Tests/UpdateResumeCoreTests.cs

```diff
@@ -42,8 +44,16 @@ public static List<Skill> GetResume()
         [Fact]
         public void UpdateResume_ExistentId_Resume()
         {
-            this.mockResumeRepository.Setup(repository => repository.UpdateResumeSkill(It.IsAny<int>(), It.IsAny<Skill[]>())).Returns(UpdateResumeCoreTests.GetResume());
-            var result = this.resumesService.UpdateResumeSkill(1, UpdateResumeCoreTests.GetResume().ToArray());
+            this.mockResumeRepository.Setup(repository => repository.UpdateResumeSkill(It.IsAny<int>(), It.IsAny<Skill[]>())).Returns(this.GetSkills());
+            var result = this.resumesService.UpdateResumeSkill(2, this.GetSkills().ToArray());
+            Assert.IsType<Skill[]>(result.ToArray());
+        }
+
+        [Fact]
+        public void UpdateResume_UnexistentId_ValidationException()
```

Comment:
this does not thrown an exception anymore it just returns null please change the name